src/main/java/se/sundsvall/emailreader/service/EmailScheduler.java

```diff
@@ -108,6 +108,7 @@ private void handleMessages(final CredentialsEntity credentials, final List<Emai
 				if (emailMap.get("Recipient") == null || emailMap.get("Message") == null) {
 					final var recipient = emailMap.get("Recipient");
 					final var message = emailMap.get("Message");
+					dept44HealthUtility.setHealthIndicatorUnhealthy(smsJobName, "Recipient or Message is missing in email");
```

Comment:
Maybe we should create consumers in constructor for all calls to setHealthIndicatorUnhealthy? One for sms-job and one for email-job. Now we are mixing consumer and direct usage (nothing technically wrong with that I guess), but we could use consumers everywhere. We are also creating consumers in a loop which seems unnecessary. In that case name should be apparent something like "emailSetUnHealthyConsumer". Thoughts?